app/controllers/health/participation_forms_controller.rb

```diff
@@ -114,7 +104,7 @@ def set_blank_form
       @blank_participation_form_url = GrdaWarehouse::PublicFile.url_for_location 'patient/participation'
     end
 
-    def form_url(opts={})
+    def form_url(_opts = {})
```

Comment:
Generally no, but I think form_url may be a drop-in replacement for a built-in method.  I'm leaving it for now because tracking down exactly which instances use it may be fairly difficult.